queries/smali/highlights.scm

```diff
@@ -0,0 +1,110 @@
+; Types
+
+(class_identifier) @type
+
+(primitive_type) @type.builtin
+
+(array_type
+  "[" @punctuation.bracket)
+
+; Methods
+
+(method_declaration
+  (method_identifier) @function)
+
+((statement) @function.call
+ (#lua-match? @function.call "^invoke"))
```

Comment:
isn't `invoke` a keyword? Could be `@keyword.function` although `@keyword.function` was only used until now for keywords introducing a function definition. `@keyword` would probably more general

Reply:
This doesn't apply to `invoke-xyz` but to the function it calls.
I guess I could specifically target the function node though.